### ISSUE TITLE
Switch to default export

### ISSUE THREAD

**Issue by mtlewis** [schibsted/jsx-pdf#15]
Right now we have a named export `createElement`, which needs to be imported in order for the jsx transform to work correctly:
```javascript
import { createElement } from 'jsx-pdf';
```
I think it would be nice to switch to exporting a single object. This allows people to use a more familiar syntax when importing our package:
```javascript
import JsxPdf from 'jsx-pdf';
```

**Comment by yaroslav-borcheninov** [comment]
I think it works. People will still be able to import the functions separately if they prefer, right?

**Comment by mtlewis** [comment]
Closed in #17.
